libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -434,6 +440,12 @@ class MapboxNavigation(
         roadObjectMatcher = RoadObjectMatcher(navigator)
     }
 
+    /**
+     * Control the location events playback during a replay trip session.
+     * Start a replay trip session with [startReplayTripSession].
+     */
+    val mapboxReplayer: MapboxReplayer by lazy { tripSessionLocationEngine.mapboxReplayer }
```

Comment:
Should this be tagged with experimental as well?

Reply:
Unfortunately, you're right 😄 .. https://github.com/mapbox/mapbox-navigation-android/pull/4856